MM-32227 - Update docs for saved messages (#4298)
mattermost/docs: source/help/messaging/flagging-messages.rst

Saving Messages
---------------

> Saving a message helps you mark messages for follow up. Your saved messages are personal, and cannot be seen by other users. There is no limit to the number of posts you can save.

Reviewer: ```suggestion
Saving a message helps you mark messages for follow up. Your saved messages are personal, and can't be seen by other users. There's no limit to the number of posts you can save.
```

> One way we like to use saved messages is to mark messages we want to respond to later. It's useful when someone asks for something that requires work. We save the post, keep doing what we're doing, and then go back to our list of **Saved Posts**. After the work is completed, we can reply back to the post and remove it from the list.

Reviewer: ```suggestion
One way to use saved messages is to mark messages you want to respond to later. You can save the post in the list of **Saved Posts**, refer back to it when you're able to, and once complete you can remove it from the list.
```

Saving a Message
=================

To try it out, hover over a message to show the **Save** icon. Click the icon to save the message for follow up.

.. image:: ../../images/save-message.png

Viewing Saved Messages

> =======================

Reviewer: ```suggestion
=====================
```

> To view a list of your **Saved Posts**, click the **Save** icon at the top of the channel.

Reviewer: ```suggestion
To view a list of your **Saved Posts**, select the **Save** icon at the top of the channel.
```

.. image:: ../../images/saved-posts.png

Unsaving Messages
=================

> To remove an item from the **Saved Posts** list, click the **Save** icon beside a message to clear it.

Reviewer: ```suggestion
To remove an item from the **Saved Posts** list, select the **Save** icon next to message to clear it.
```
